index.html

```diff
@@ -11,23 +11,34 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/cc9d26c8d1.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="src/on-event.js" defer></script>
     <title>mobile version</title>
   </head>
 
   <body>
     <header class="header">
       <a class="container" href="index.html">Arrow Dan</a>
       <nav class="navbar">
-        <a href="#"
-          ><img
+        <div>
+          <img
             class="hamburger hide-desktop"
             src="image/Union.png"
             alt="navigation icon"
-        /></a>
+          />
+        </div>
+
         <ul class="hide-mobile">
-          <li><a href="">Portfolio</a></li>
-          <li><a href="">About</a></li>
-          <li><a href="">Contact</a></li>
+          <li class="nav-links" id="portfolio">
+            <a href="#project-section">Portfolio</a>
+          </li>
+          <li class="nav-links" id="about"><a href="#about-me">About</a></li>
+          <li class="nav-links" id="contact">
+            <a href="#contact-section">Contact</a>
+          </li>
```

Comment:
- [ ] Kindly make sure that the mobile menu options take you to the correct page when clicked.

https://user-images.githubusercontent.com/70269507/167641396-1fd1ffaa-c6ef-48f9-97ae-7082eb2e707e.mov